.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,31 +1,29 @@
 ## Proposed changes
 
-Close related #issue (replace it with issue number if it exists).
+Close #issue (replace it with issue number if it exists).
 
 Describe the overview of changes, and introduce why we need it.
 
 ## Types of changes
```

Comment:
I think we should remove `Types of changes`, because it's duplicated with the commit tags.

Reply:
> I think we should remove `Types of changes`, because it's duplicated with the commit tags.

I also feel that.

Reply:
What about guiding contributes to completing the commit tags via http://doris.incubator.apache.org/master/zh-CN/community/commit-format-specification.html here?